doc/EPIC.md

```diff
@@ -0,0 +1,110 @@
+# EPIC Design 
+
+This file introduces EPIC and documents the design rationales and best practices for EPIC-HP.
+
+## Introduction
+One important security property of SCION is that end hosts are not allowed to send packets along arbitrary paths, but only along paths that were advertised by all on-path ASes. 
+This property is called *path authorization*. 
+The ASes only advertise paths that serve their economic interests, and path authorization protects those routing decisions against malicious end hosts.
+
+In SCION, this is implemented by having the ASes create authenticators during beaconing, which end hosts then have to include in the MAC field of their packets. Those MAC fields prove to the ASes, that the packets are allowed to traverse them.
+The MAC fields are static, meaning that they are the same for every packet on that path.
+
+However, this implementation of path authorization is insufficient against more sophisticated adversaries that are able to derive MAC fields, for example by brute-forcing them or by observing them in the data plane packets. Because the MACs are static, once derived MACs for one packet can be reused by the adversary to send arbitrarily many other packets (until the authenticators expire).
```

Comment:
I wouldn't include "observe" in "derive". Also, while we argued about brute-forcing non-authorized paths in the EPIC paper, here observing authorized but hidden paths is more relevant, so I would focus on that.
This applies to the rest of this document as well.

Reply:
Thank you, I changed it accordingly.